spec/system/coupon_code_spec.rb

```diff
@@ -80,45 +80,6 @@ def create_basic_coupon_promotion(code)
           end
         end
       end
-
```

Comment:
I think this needs an explanation (maybe just in the commit description). Why it's ok to remove this spec?

Reply:
It's testing 2 unrelated things (coupon code submission and payment method selector) that are being already tested separately in their respective specs. I'll add the explanation in the commit description. 👍